applications/spring-shell/pom.xml

```diff
@@ -99,6 +99,12 @@
             <version>1.16.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Hm, I think we need this dependency for the tests for now? 
Would prefer not to have them in here. But will move them out when e2e tests are provided for each module.
But could you please move the `amqp-client` dependency down here and set the scope to `test`? 
And add a comment mentioning that these dependencies are needed by the tests for mule-migration?